packages/ui/src/helpers/authenticator/formFields/defaults.ts

```diff
@@ -45,9 +45,7 @@ export const getAliasDefaultFormField = (
 };
 
 /** Reusable confirmation code form fields. */
-const getConfirmationCodeFormFields = (
-  state: AuthMachineState
-): FormFields => ({
+const getConfirmationCodeFormFields = (_: AuthMachineState): FormFields => ({
```

Comment:
Linting fix due to new rules ([ref](https://github.com/aws-amplify/amplify-ui/pull/5830/files#diff-6e97cd3294540e00be8979d5d20d4f092e2d49d3d7e796d472f3e03961d69ab5R31))